ComposeSample/.idea/jarRepositories.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think Cesar told us that the entire .idea folder should be added to the git.ignore file